geoh5py/ui_json/validation.py

```diff
@@ -264,7 +264,8 @@ def validate(
 
     def validate_data(self, data: dict[str, Any]) -> None:
         """
-        Calls validate method on individual key/value pairs in input.
+        Calls validate method on ui.json data structure for cross-dependencies,
```

Comment:
Should be a one-liner followed by more details

Reply:
Done